### DISCUSSION TITLE
I need help with this error

### DISCUSSION THREAD

**Discussion by SamLiftsWeight** [winstxnhdw/lc-hax#482] [Q&A]
So I have used these hacks many times before and I went to load them up when i got this error: Windows cannot find './submodules/SharpMonoInjectorCore/dist/SharpMonoInjector.exe'. Make sure you typed the name correctly. It also said in the launch 1 of 2 projects up to date so please help me.

**Comment by winstxnhdw** [comment]
Run requirements again. I guess your antivirus removed it.

**Comment by SamLiftsWeight** [comment]
Bro its not working

**Reply by winstxnhdw** [reply]
Did you miss the part where I told you its an antivirus issue?

**Comment by SamLiftsWeight** [comment]
Yeah its windows security but when i went in to turn it off there is no virus protection section i even looked up how to do it and its literally missing

**Reply by winstxnhdw** [reply]
Run requirements while looking at it

**Reply by SamLiftsWeight** [reply]
alr

**Reply by SamLiftsWeight** [reply]
I am first deleting it all then ima redownload it and try doing that

**Comment by SamLiftsWeight** [comment]
Um when i redownloaded git and typed the code in it said this: sgoer@SamsBoner MINGW64 ~
$ git clone --recursive --depth 50 https://github.com/winstxnhdw/lc-hax.git
fatal: destination path 'lc-hax' already exists and is not an empty directory. Is that bad?

**Reply by winstxnhdw** [reply]
yeah u gotta delete it

**Reply by SamLiftsWeight** [reply]
delete what

**Reply by winstxnhdw** [reply]
the lc-hax folder?

**Reply by SamLiftsWeight** [reply]
oh alr

**Reply by SamLiftsWeight** [reply]
Ok so im a bit pist, when i redownloaded all of it and did requiredments and then i went to inject it said this: The system cannot execute the specified program. then my anti virus message popped up and when i clicked on it, it said Page not available. Your IT administrator has limited access to some areas of this app, and the item you tried to access is not available. Contact IT helpdesk for more info

**Reply by winstxnhdw** [reply]
Why are you not admin on your own computer?

**Reply by SamLiftsWeight** [reply]
IDK

**Reply by winstxnhdw** [reply]
and I am supposed to know? I can’t help you if you can’t do even the bare minimum for a Windows user.
